docs/release_notes/0.143.0.md

```diff
@@ -0,0 +1,13 @@
+# Release v0.143.0
+
+## 🚀 Features
+
+- Add support for EKS 1.27 (#6639)
+
+## 🎯 Improvements
+
+- Enforce IMDSv2 by default (#6631)
+
+## 🐛 Bug Fixes
+
+- Fix readiness probe in HTTP test (#6628)
```

Comment:
I'd put this under maintenance section as it's not a user facing bug, but a syntax error in an integration test.